src/pages/Landing/index.tsx

```diff
@@ -174,16 +174,16 @@ const SubText = styled.div`
   }
 `
 
-const SubTextContainer = styled.div<{ $visible: boolean }>`
+const SubTextContainer = styled.div<{ initiallyVisible: boolean; applyFadeIn: boolean }>`
   display: flex;
   justify-content: center;
-
-  ${({ $visible }) =>
-    $visible
-      ? css`
-          ${textFadeIn}
-        `
-      : 'opacity: 0;'}
+  ${({ initiallyVisible }) => (initiallyVisible ? `opacity: 1.0;` : 'opacity: 0.0;')}
+  ${({ applyFadeIn }) =>
+    applyFadeIn &&
+    css`
+      ${textFadeIn}
```

Comment:
won't the fade in only occur if the opacity was not initially 1.0? in this case, is it necessary for applyFadeIn to be used? Couldn't it just always be applied?

Reply:
the fade in should not occur when the origin country is available on the first render only if it is being fetched later. 
so if its available on the first render i set the opacity to 1.0 and the apply fade in effect will not happen